go.mod

```diff
@@ -813,9 +813,21 @@ require (
 )
 
 require (
+	github.com/influxdata/influxdb-client-go/v2 v2.14.0
+	github.com/influxdata/influxdb1-client v0.0.0-20220302092344-a9ab5670611c
+	github.com/open-telemetry/opentelemetry-collector-contrib/receiver/influxdbreceiver v0.108.0
```

Comment:
would be better to put the requirement next to the other otel components requirements to keep them grouped